File: PokerBomb-Server/PokerBomb-hall/src/main/java/module/pokerbomb/to/GameRole.java
package module.pokerbomb.to;

import lombok.Data;

import java.util.List;

@Data
public class GameRole {
    private Integer userId;

    private String userName;

    private String userIcon;

    private Integer totalPoints; //总分

    private Integer juPoints;  //局分

    private Integer panPoints; //盘分

    private UserStatus userStatus;

    private List<Integer> userCards;
}

File: PokerBomb-Client/assets/Script/common/GameRole.ts
import { _decorator, Component, Node } from 'cc';
const { ccclass, property } = _decorator;

@ccclass('GameRole')
export class GameRole extends Component {
    public userId;

    public userName;

    public userIcon;

    public totalPoints;

    public juPoints;

    public panPoints;

    public userCards;

    public userStatus;
    
    start() {

    }

    update(deltaTime: number) {
        
    }
}

